composer.json

```
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": "^7.4|^8.0",
+        "php": "^7.4|^8.1",
```

Hi @AndreyPootMay :wave: Thank you very much for the contribution :tada::tada::tada: 

Check the following suggestion, only allowing PHP ^7.4|^8.1 will block installing with PHP 8.0
```suggestion
        "php": "^7.4|^8.0|^8.1",
```